### ISSUE TITLE
cmd/config: --config flag loading bug

### ISSUE THREAD

**Issue by buchanae** [ohsu-comp-bio/funnel#67]
```
../../bin/funnel server -config funnel.config.yml
INFO tes        Using config file               path=/Users/buchanae/projects/funnel/deployments/gce/onfig
ERRO tes        Failure reading config          error=open /Users/buchanae/projects/funnel/deployments/gce/onfig: no such file or directory path=/Users/buchanae/projects/funnel/deployments/gce/onfig
```

**Comment by buchanae** [comment]
Oh, this is due to `--config` vs `-config`, which is maybe ok if the error message is clear.

**Comment by adamstruck** [comment]
Yeah.. This is because `--config` is aliased as `-c` so when you write `-config` it reads in `onfig` as the argument to the flag.

I wonder if we can configure the flag parsing to require a separator.

**Comment by buchanae** [comment]
Maybe it's worth not having a short version for config. Also, I think the absolute path made things a little more confusing.

**Comment by adamstruck** [comment]
closed by #372 

Additional validation by cobra will help avoid this type of confusion.
